tests/test_ops/test_bbox.py

```diff
@@ -2,41 +2,49 @@
 import pytest
 import torch
 
+from mmcv.ops import bbox_overlaps
+
 
 @pytest.mark.skipif(
     not torch.cuda.is_available(), reason='requires CUDA support')
 class TestBBox(object):
 
-    def _test_bbox_overlaps(self, dtype=torch.float):
+    def setup_class(self):
+        self.b1_1 = torch.tensor([[1.0, 1.0, 3.0, 4.0], [2.0, 2.0, 3.0, 4.0],
```

Comment:
`b1_1` is not a proper name.
`bbox_1_1` maybe better.

Reply:
done